src/fsharp/FSharp.Compiler.Service/StandardError.txt

```diff
@@ -0,0 +1,7 @@
+Could not execute because the application was not found or a compatible .NET SDK is not installed.
```

Comment:
Are these files intented to be commited? Shall we add them to the .gitignore?

Reply:
Removed the file, thanks (yes should add to